config/foreman.hiera/scenario/katello.yaml

```diff
@@ -0,0 +1,2 @@
+---
+puppet::server::foreman::katello: true
```

Comment:
So I fully understand, this will only trigger this class if the main puppet class is enabled?

Reply:
Pretty much, but I'm going to be pedantic.

Technically only if the `puppet::server::foreman` class ends up being evaluated without a more specific answer. Hiera is more like a datastore. If Puppet doesn't ask about the value, it doesn't affect anything.

For example, this will trigger a lookup:
```puppet
include puppet::server::foreman
```

But this won't:
```puppet
class { 'puppet::server::foreman':
  katello => true, # or false - doesn't matter
}
```

For us that typically means you need to: `include puppet`, make sure both `puppet::server` and `puppet::server_foreman` are true.